### ISSUE TITLE
[Feature - Sparrow Server] Preserve mix counts during mix-out

### ISSUE THREAD

**Issue by RequestPrivacy** [sparrowwallet/sparrow#1013]
## Feature Request
If someone mixes out from -> to wallets managed by the same Sparrow Server, mix numbers/counts aren't transferred to the UTXO tab of the receiving wallet. I guess since Sparrow Server is managing both wallets & mix processes it could somehow preserve the mix count between the wallets.

## Reasoning
The number one differentiator between utxos in postmix is the number of rounds they've been whirlpooled. If users whirlpool with sparrow server back-end and want to transact (meaning pay someone) e.g. with Sparrow Desktop ideally they want to choose the coins with the most amount of mixes received. A "re-setted" mix count makes this harder. Following the mix process in the GUI to identify the coins which have currently the max amount of mixes isn't ideal and even harder if the mixing is done across multiple wallets (e.g. for keeping the depth of a wallet reasonable).

If **Sparrow Server** would retain the mix amount between mix-outs to Sparrow Server wallets one could easily identify the coins with most mixes in Sparrow Server and via confirmation time or other identifiers match these coins in the GUI where the final payment is constructed (as the retained mix count is only possible in the Server instance, not the GUI. Ideally there would also be information flow from Sparrow Server -> Sparrow GUI and vice versa, but I guess that's much harder)

**Comment by craigraw** [comment]
> The number one differentiator between utxos in postmix is the number of rounds they've been whirlpooled

This is actually not really true - the real differentiator is the anonymity set, which depends on the mix counts of the peers as well. Unfortunately it's too computationally intensive to calculate this, so the mix count is a substitute. 

Have you tried refreshing the wallet in Sparrow Desktop? This will clear all UTXOs, reload them from the server, and recalculate mix counts.

**Comment by RequestPrivacy** [comment]
> This is actually not really true - the real differentiator is the anonymity set 

Right, I was being sloppy/wrote from the perspective of a user who sees the mix count as "the" number.

> Have you tried refreshing the wallet in Sparrow Desktop?

Good point for the GUI, although it feels rather hacky/workaround. What about the terminal and the described mix-out to a wallet managed by the same Sparrow Terminal?
